corehq/apps/app_manager/tests/data/v2_diffs/partials/case_config.html.diff.txt

```diff
@@ -31,9 +31,7 @@
  
  <div id="case-config-ko">
      <div data-bind="saveButton: saveButton"></div>
-@@ -70,71 +85,77 @@
-         <div data-bind="template: 'case-config:refresh-form-questions'"></div>
-     {% endif %}
```

Comment:
Looks like an instance in which this test was a "false positive"—i.e. made the build fail, but where the code itself was fine and required a change only in the test. Nothing actionable, just noting here, since if that happens to often it dilutes the signal and could become error-prone. My guess is it's a good-enough solution and worth being super-simple to implement.